### PR TITLE
New UI - Edit group page and New group page

### PR DESCRIPTION
**Pull request by mohit-003**
image
image

### CODE REVIEW

**Comment by nitin10s** [comment]
Decrease the margin between Save button and hr

**Comment by nitin10s** [comment]
Increase the width of the text bo in New Group Section, and slightly increase the Y-Margins of the text box

**Comment by Ananthavijay** [comment]
Shouldn't there be a ````back button```` on the new group's page?

**Comment by nitin10s** [comment]
Yes, you missed the back button as well. The text "save" should be replaced to Make New Group and the "back" button should be put at the right side of the button without any bg, just the text with an underline. Thanks, @Ananthavijay :)

File: app/assets/stylesheets/groups.scss [hunk 1]
```diff
@@ -84,12 +80,8 @@
   color: $white;
 }
 
-.btn-delete {
-  border: 1px solid $danger;
-}
-
-.btn-delete:hover {
-  background-color: $danger;
+.btn-del:hover {
```
Comment (tachyons) [inline app/assets/stylesheets/groups.scss]: Use sass pseudo class syntax
```scss
.btn-del {
  &:hover { }
}
```

File: app/assets/stylesheets/logixRelease.scss [hunk 2]
```diff
@@ -6,9 +6,21 @@
...
-    border: 1px solid rgba(0, 0, 0, 0.125);
+    border-radius: .25em;
+    height: 200px;
+    margin-bottom: 20px;
+    margin-top: 40px;
     padding: 1.25rem;
-    border-radius: 0.25em;
+}
+
+.field{
```
Comment (tachyons) [inline app/assets/stylesheets/logixRelease.scss]: Can't just use `center-block` class which is already available in bootstrap

File: app/assets/stylesheets/logixRelease.scss [hunk 3]
```diff
@@ -281,3 +293,42 @@ a.btn.btn-outline-danger {
 .subheading_teachers{
   text-align: left;
 }
+.edit_name{
```
Comment (tachyons) [inline app/assets/stylesheets/logixRelease.scss]: Prefer hyphen instead of  underscore for class names

File: app/views/groups/_form.html.erb [hunk 4]
```diff
@@ -2,22 +2,18 @@
...
       <% end %>
       </ul>
     </div>
   <% end %>
-
-  <div class="field form-group">
-    <%= form.label :name %>
-    <%= form.text_field :name, id: :group_name, class: 'form-control' %>
+  <div class="field">
+   <label for="group_group_name" class="name-group">GROUP NAME:</label>
```
Comment (tachyons) [inline app/views/groups/_form.html.erb]: Use label helper https://apidock.com/rails/ActionView/Helpers/FormHelper/label

File: app/views/groups/edit.html.erb [hunk 5]
```diff
@@ -1,23 +1,15 @@
...
   </div>
-  <br />
-
-  <br />
-
-
-
+  <hr class="w-75">
+  <div class="button-group">
+   <button class="btn-view"> <svg class="pr-1" xmlns="http://www.w3.org/2000/svg" width="20" height="20" viewBox="0 0 24 24"> <path d="M15 12c0 1.654-1.346 3-3 3s-3-1.346-3-3 1.346-3 3-3 3 1.346 3 3zm9-.449s-4.252 7.449-11.985 7.449c-7.18 0-12.015-7.449-12.015-7.449s4.446-6.551 12.015-6.551c7.694 0 11.985 6.551 11.985 6.551zm-7 .449c0-2.761-2.238-5-5-5-2.761 0-5 2.239-5 5 0 2.762 2.239 5 5 5 2.762 0 5-2.238 5-5z"/> </svg>  <%= link_to 'View', @group, class: 'text-dark' %></button>
```
Comment (tachyons) [inline app/views/groups/edit.html.erb]: - Use button tag
- btn-view is generic, are we using same style across cv ?

**Pull request merged**